deployment/ccip/changeset/v1_6/cs_ccip_home.go

```diff
@@ -1416,3 +1430,129 @@ func ValidateCCIPHomeConfigSetUp(
 	}
 	return nil
 }
+
+type DeployDonIDClaimerConfig struct {
+	HomeChainSelector uint64 `json:"homeChainSelector"`
```

Comment:
nit: we decided that instead of asking user for homechain selector, we will derive it from https://github.com/smartcontractkit/chainlink/blob/develop/deployment/ccip/changeset/state.go#L665 - will this work here?

Reply:
yup derived here: https://github.com/smartcontractkit/chainlink/pull/17196/commits/3032915a8960184cd07d4026341875e5446e27f6